test/box/gh-6292-fix-dropping-msg-when-connection-closing.test.lua

```diff
@@ -0,0 +1,58 @@
+net = require('net.box')
+fiber = require('fiber')
+test_run = require('test_run').new()
+
+box.schema.user.grant('guest', 'execute', 'universe')
+old_readahead = box.cfg.readahead
+old_net_msg_max = box.cfg.net_msg_max
+
+--
+-- Check that tarantool process all requests when connection is closing,
+-- net_msg_max limit is reached and readahead limit is not reached:
+--
+box.cfg ({ net_msg_max = 2, readahead = 4 * old_readahead })
+counter = 0
+expected_counter = old_net_msg_max * 5
```

Comment:
Shouldn't you use box.cfg.net_msg_max here?

Reply:
It's better to use big value, like 1024, because, if it's small, all requests can be processed before connection closed.